llvm/test/Transforms/Util/loop-iter-count-asserts.ll

```diff
@@ -0,0 +1,295 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py
+; NOTE: Example file for converting loop iter count to assumes in Loop
```

Comment:
nit: move this note to after the copyright notice.

Reply:
moved it down